setup.py

```diff
@@ -52,6 +52,8 @@
         'djangorestframework==3.3.2',
         'djangorestframework-filters==0.8.0',
         'django-autoslug>=1.9.0',
+        # XXX: fails with the latest django-filter 0.14.0
```

Comment:
Add a note that `djangorestframework-filters` is failing with the newest version and this line should be completely removed when fixed.